misc/specs/atrium-specs-jvm/src/main/kotlin/ch/tutteli/atrium/specs/reporting/ObjectFormatterSpec.kt

```diff
@@ -9,32 +9,27 @@ import ch.tutteli.atrium.reporting.StringBasedRawString
 import ch.tutteli.atrium.reporting.translating.TranslatableBasedRawString
 import ch.tutteli.atrium.reporting.translating.Translator
 import ch.tutteli.atrium.specs.AssertionVerb
-import ch.tutteli.atrium.specs.describeFun
-import com.nhaarman.mockitokotlin2.any
-import com.nhaarman.mockitokotlin2.doReturn
-import com.nhaarman.mockitokotlin2.mock
-import org.jetbrains.spek.api.Spek
-import org.jetbrains.spek.api.dsl.SpecBody
-import org.jetbrains.spek.api.dsl.context
-import org.jetbrains.spek.api.dsl.it
-
-//TODO #116 migrate spek1 to spek2 - move to specs-common
+import ch.tutteli.atrium.specs.describeFunTemplate
+import io.mockk.*
+import org.spekframework.spek2.Spek
+import org.spekframework.spek2.style.specification.Suite
+
 abstract class ObjectFormatterSpec(
     testeeFactory: (Translator) -> ObjectFormatter,
     describePrefix: String = "[Atrium] "
 ) : Spek({
 
-    fun describeFun(vararg funName: String, body: SpecBody.() -> Unit) =
-        describeFun(describePrefix, funName, body = body)
+    fun describeFunTemplate(vararg funName: String, body: Suite.() -> Unit) =
```

Comment:
```suggestion
    fun describeFun(vararg funName: String, body: Suite.() -> Unit) =
```

Reply:
Hi, thanks for your feedback
Originally there were 'on' blocks which need to be changed to 'context' - we might want to log this as a rule of thumb for later changes.

Another issue I face is nesting 'group' block under 'describe'
compilation fails, which can be resolved in one of two ways:
- change group to context
- move group block outside of containing describe

what do you think?

Compilation error is: 'fun group(description: String, skip: Skip = ..., defaultCachingMode: CachingMode = ..., preserveExecutionOrder: Boolean = ..., failFast: Boolean = ..., body: GroupBody.() -> Unit): Unit' can't 
be called in this context by implicit receiver. Use the explicit one if necessary

Reply:
> Originally there were 'on' blocks which need to be changed to 'context' - we might want to log this as a rule of thumb for later changes.

Good idea, I have updated the description of the issue (also the fact that `context` should be used instead of `group`)